tests/fixtures/semantic_parsing/nlvr_coverage_semantic_parser/serialization/vocabulary/non_padded_namespaces.txt

```diff
@@ -0,0 +1,2 @@
+actions
```

Comment:
You may want to regenerate the fixtures though. This seems outdated. The non-padded namespaces should be `denotations` and `rule_labels`. I fixed that in the config files recently.